webroot/results/person.php

```diff
@@ -66,6 +66,17 @@ function showBody () {
   if( $picture )
     echo "<center><img class='person' src='$picture' /></center>";
 
+  #--- Show the In Memoriam if any.
+  $inMemoriamArray = array(
+    "2008COUR01" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=2028",
+    "2003LARS01" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=1982",
+    "2012GALA02" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=1044",
+    "2008LIMR01" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=945",
+    "2008KIRC01" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=470",
+  );
+  if( array_key_exists($chosenPersonId, $inMemoriamArray) )
```

Comment:
Could you use curly braces here? I personally am not a fan of ever ommitting them (the classic example is that someone goes and adds a line of code and doesn't realize it's no longer in the if statement).